paimon-format/src/main/java/org/apache/paimon/format/orc/OrcReaderFactory.java

```diff
@@ -268,7 +270,12 @@ private static RecordReader createRecordReader(
                             .skipCorruptRecords(OrcConf.SKIP_CORRUPT_DATA.getBoolean(conf))
                             .tolerateMissingSchema(
                                     OrcConf.TOLERATE_MISSING_SCHEMA.getBoolean(conf));
-
+            if (!conjunctPredicates.isEmpty()) {
+                // TODO fix it , if open this option,future deletion vectors would not work,
+                //  cased by getRowNumber would be changed .
+                options.useSelected(OrcConf.READER_USE_SELECTED.getBoolean(conf));
```

Comment:
I am very concerned that this may affect some scenarios, such as the user's built-in Hadoop configuration containing these configurations, which may be transmitted here.

Can we create a new option to enable these two options?